lib/task_screen.dart

```diff
@@ -54,6 +54,7 @@ class TaskScreen extends StatelessWidget {
       );
     }
 
+    // Carlo
```

Comment:
Possibly simplify the navigation to the TaskDetailsPage by extracting it into a separate function.

Possibly simplify the creation of ListTile widgets by using the map function.